lib/rules/function-url-no-scheme-relative/index.mjs

```diff
@@ -41,6 +41,27 @@ const rule = (primary) => {
 				});
 			});
 		});
+
+		root.walkAtRules('import', (atRule) => {
+			const params = atRule.params;
+
+			functionArgumentsSearch(params.toLowerCase(), 'url', (args, index) => {
+				const url = args.trim().replace(/^['"]+|['"]+$/g, '');
+
+				if (!isStandardSyntaxUrl(url) || !url.startsWith('//')) {
+					return;
+				}
+
+				report({
+					message: messages.rejected,
+					node: atRule,
+					index,
+					endIndex: index + args.length,
+					result,
+					ruleName,
+				});
+			});
```

Comment:
refactor
minor

check l27–42
i.e. you could reuse the same function
e.g.
```js
const foo = (node) =>  (args, index) => {
	const url = args.trim().replace(/^['"]+|['"]+$/g, '');

	if (!isStandardSyntaxUrl(url) || !url.startsWith('//')) {
		return;
	}

	report({
		message: messages.rejected,
		node,
		index,
		endIndex: index + args.length,
		result,
		ruleName,
	});
}
```

Reply:
[Note] Such functions are often named `check` in other rules